packages/@o3r/training-sdk/tsconfig.json

```
@@ -0,0 +1,14 @@
+/* IDE usage only */
+{
```

missing base extension
```suggestion
{
  "extends": "../../../tsconfig.base",
```

---

I will update the file, this line is not present in the tsconfig.json of the showcase-sdk, should we update this as well?